docs/Makefile

```diff
@@ -18,3 +27,6 @@ image: ## Build a container image for the preview of the website
 clean:
 	docker image rm $(CONTAINER_IMAGE)
 
+.PHONY: tracing-policy-docs
+tracing-policy-docs: ## Generate Tracing Policy reference page from CRDs.
```

Comment:
could you put a reference to this in the main Makefile using something like:
```Makefile
.PHONY: tracing-policy-docs
tracing-policy-docs:
  $(MAKE) -C docs tracing-policy-docs
```

Reply:
Added in 4b72dfbd877ca4e9b5edd97930fe71aa76ee6e93